modules/images/webp-uploads/settings.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Settings for the webp uploads module.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Register media settings field to generate multiple mime WebP & JPEG output.
+ *
+ * @since n.e.x.t
+ */
+function webp_uploads_register_media_settings_field() {
+	// Register field setting for the processing.
+	register_setting( 'media', 'perflab_generate_webp_and_jpeg' );
```

Comment:
I think my comment in https://github.com/WordPress/performance/pull/537#discussion_r988060196 wasn't precise enough, I meant to "fully" register the setting, i.e. also including the `$args` parameter.
```suggestion
	register_setting(
		'media',
		'perflab_generate_webp_and_jpeg',
		array(
				'type'         => 'boolean',
				'default'      => false,
				'show_in_rest' => false,
		)
	);
```

Reply:
The changes make sense.